ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -135,6 +139,27 @@ class CommandBarButton: UIButton {
                         dark: Colors.primary(for: window))
     }
 
+    private var accentImageView: UIImageView?
+
+    private func updateAccentImage(accentImage: UIImage?) {
+        if accentImage != accentImageView?.image {
+            if let accentImage = accentImage {
+                accentImageView = UIImageView(image: accentImage)
+                accentImageView?.translatesAutoresizingMaskIntoConstraints = false
+                if let accentImageView = accentImageView, let imageView = imageView {
```

Comment:
minor: for clarity and debuggability, usually good to put separate `if let` clauses on separate lines